release-tools/RELEASE.md

```diff
@@ -0,0 +1,5 @@
+# Release Process
+
+No tagged releases are planned at this point. The intention is to keep
```

Comment:
Should this have the explanation on how to use this, or is it the README.md that we should follow?